src/styles/getMuiTheme.js

```diff
@@ -224,4 +226,12 @@ export default function getMuiTheme(baseTheme, muiTheme) {
       borderColor: palette.borderColor,
     },
   }, muiTheme);
+
+  if (computedTheme.isRtl) {
```

Comment:
@alitaheri How do you feel about this?

The only reason why this is slightly different than before is that the `rtl` transformer will be added to the list of transforms is `isRtl` is set to true (regardless of whether or not the specified their own custom list of transformers). I think I'm okay with this because `isRtl` is false by default.

Also, I'm thinking that if we make this a documented API we should eventually deprecate `isRtl` in favor of them just applying the transformer themselves.